scripts/world_bank/wdi/worldbank.py

```diff
@@ -272,7 +425,7 @@ def download_indicator_data(worldbank_countries, indicator_codes,
         country_df['ISO3166Alpha3'] = country_code
 
         # Add new row to main datframe.
-        worldbank_dataframe = worldbank_dataframe.append(country_df)
+        worldbank_dataframe = pd.concat([worldbank_dataframe, country_df])
```

Comment:
Instead of appending for each country, collect all country data frames into an array and concat once after the loop. This will speedup processing.

Reply:
Modified as suggested.